cclient/api/core/core_init.c

```diff
@@ -8,12 +8,29 @@
 #include "cclient/api/core/core_init.h"
 #include "cclient/api/core/logger.h"
 
-retcode_t iota_client_core_init(iota_client_service_t* const serv) {
-  logger_init_client_core();
-  return iota_client_service_init(serv);
+static char const *client_path = "/";
+static char const *client_content_type = "application/json";
+
+iota_client_service_t *iota_client_core_init(char const *host, uint16_t port, char const *ca_pem) {
```

Comment:
I'd personally not call this init since it doesn't initialise but creates a new service

Reply:
I'm okay with it.